### PR TITLE
New UI Design (step 1)

### PR DESCRIPTION
**Pull request by TG908**
The new UI is ready so far. But I need your feedback for the last refinements.

**Things todo:**
- [x] Reload cards when order has been edited
- [x] Fix Tum Logo   
- [x] Fix add new cards button

First we need to implement these changes (step 1). I will go ahead and create another PR with: some improvements to the model (step 2), better looking cards (step 3) and new cards (step 4).

**Interesting thing to note: The Book Rentals still work!!**

### CODE REVIEW

**Comment by mammuth** [comment]
Sweet, I like the first impression!

File: TUM Campus App/CafeteriaViewController.swift [hunk 1]
```diff
@@ -180,6 +178,8 @@ extension CafeteriaViewController: UITableViewDataSource, UITableViewDelegate {
         }
         let item = categories[indexPath.section].1[indexPath.row]
         let cell = tableView.dequeueReusableCell(withIdentifier: item.getCellIdentifier()) as? CardTableViewCell ?? CardTableViewCell()
+        cell.color1 = .white
```
[the remark this answers is not in the document]
Comment (TG908) [reply TUM Campus App/CafeteriaViewController.swift]: it ll be top and bottom gradient color then...

**Comment by TG908** [comment]
@mammuth changes in podfile are needed since I included a hot-fix for Sweeft see #159

**Comment by mammuth** [comment]
> First we need to implement these changes. I will go ahead an create another PR with: Better looking cards and new cards

I'm not quite understanding what you're planning to do. As far as I see, this PR doesn't remove any features, does it? It "only" updates the UI for the cards?

In that case - let's encapsulate this change in this PR and do the mentioned other tasks in separate PR's. Especially stuff like #165 does not to be implemented within this PR imho 😊 

If the reloading after editing was not implemented before - do that in another branch. We'd like to release within the next days and I'd love to have the new cards UI in that release! ⭐️

File: TUM Campus App/CafeteriaViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (mammuth) [reply TUM Campus App/CafeteriaViewController.swift]: That's something I and the future reader can understand 👍 😊

[the remark this answers is not in the document]
Comment (mathiasquintero) [reply TUM Campus App/CafeteriaViewController.swift]: @TG908 Can you make those changes soon. We want to release this soon

**Comment by TG908** [comment]
> > First we need to implement these changes. I will go ahead an create another PR with: Better looking cards and new cards
> 
> I'm not quite understanding what you're planning to do. As far as I see, this PR doesn't remove any features, does it? It "only" updates the UI for the cards?

You are right I am not removing any feature but I want to make another change to the UI Design (step3) which could possibly brake this Branch (because it might not work as i planned it) or it could delay the new UI. This is why I am doing  it in separate steps

**Comment by mathiasquintero** [comment]
@TG908 is this ready? If so please fix the conflicts so that we can merge it.

**Comment by TG908** [comment]
@mathiasquintero step 3 kind depends on step 2 (and thus the network refactor), to make the cards look better I want to use collection views inside the cards. For that to work i really need models that contain multiple DataElements.

**Comment by mathiasquintero** [comment]
@TG908 Ok. So maybe we can release this for now and we merge the rest with the network refactor. If you're ok with this I'll adjust the tags accordingly

File: TUM Campus App/CafeteriaViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (TG908) [reply TUM Campus App/CafeteriaViewController.swift]: done

**Comment by TG908** [comment]
This PR is good to go from my side

**Comment by mathiasquintero** [comment]
Ok. Let's wait for the CI and we'll merge it

**Pull request merged**